test/helpers.spec.ts

```diff
@@ -791,7 +791,7 @@ describe('helpers', () => {
         });
 
         it('empty array returns empty array', () => {
-          const input = [];
+          const input: never[] = [];
           const length = faker.number.int({ min: 1, max: 6 });
           const unique = faker.helpers.uniqueArray(input, length);
```

Comment:
Explicit typing could be avoided by directly providing the value as an argument. 

```suggestion
          const length = faker.number.int({ min: 1, max: 6 });
          const unique = faker.helpers.uniqueArray([], length);
```